### PR TITLE
Updated footer, fix #208

### LINKED ISSUE #208: Footer updates (copyright + Linode)

### PR DESCRIPTION
**Pull request by BernardZhao**

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -7,12 +7,18 @@
       <a href="https://www.digitalocean.com">DigitalOcean</a> for sponsoring the
       VMs used in both tracks of the DeCal
     </p>
-
+    <p>
+      <a href="https://www.linode.com">
+        <img src="/images/linode.png" style="height: 50px; width: 125px;" /></a>
```
Comment (dkess) [inline _includes/footer.html:12]: I know we're not 100% consistent about this, but please try to use relative URLs. E.g. `src="{{ "/images/linode.png" | relative_url }}"`

Comment (BernardZhao) [reply _includes/footer.html:12]: Sorry im not too sure what you mean, isn't the URL already relative?

Comment (dkess) [reply _includes/footer.html:12]: If it starts with a slash it's not relative. The idea is that you should be able to host the jekyll site in a subdirectory and `relative_url` will ensure the subdirectory is included in the path.

Comment (BernardZhao) [reply _includes/footer.html]: Got it, fixed it.

**Pull request merged**